source/docs/hardware/sensors/serial-buses.rst

```diff
@@ -8,72 +8,78 @@ Both the digital and analog inputs are highly limited in the amount of data that
 Types of supported serial buses
 -------------------------------
 
-The roboRIO supports three basic types of serial communications:
-
-- `I2C`_
-- `SPI`_
-- `RS232`_
-- `USB`_
-
-Additionally, the roboRIO supports communications with peripheral devices over the CAN bus.  However, as the FRC CAN protocol is quite idiosyncratic, relatively few peripheral sensors support it (though it is heavily used for motor controllers).
+The roboRIO supports many basic types of serial communications.
 
 I2C
 ^^^
 
-|roboRIO I2C| |I2C Pinout|
+.. image:: images/roborio-ports/roborio-i2c.png
+
+.. image:: images/serial-buses/i2c-pinout.png
 
 To communicate to peripheral devices over I2C, each pin should be wired to its corresponding pin on the device.  I2C allows users to wire a "chain" of slave devices to a single port, so long as those devices have separate IDs set.
 
-The I2C bus can also be used through the `MXP expansion port`_.
+The I2C bus can also be used through the MXP Expansion Port.
 
 SPI
 ^^^
 
-|roboRIO SPI| |SPI Pinout|
+.. image:: images/roborio-ports/roborio-spi.png
+.. image:: images/serial-buses/spi-pinout.png
 
 To communicate to peripheral devices over SPI, each pin should be wired to its corresponding pin on the device.  The SPI port supports communications to up to four devices (corresponding to the CS 0-3 pins on the diagram above).
 
-The SPI bus can also be used through the `MXP expansion port`_.
+The SPI bus can also be used through the MXP Expansion Port.
 
 RS232
 ^^^^^
 
-|roboRIO RS232| |RS232 Pinout|
+.. image:: images/roborio-ports/roborio-rs-232.png
+
+.. image:: images/serial-buses/rs232-pinout.png
 
 To communicate to peripheral devices over RS232, each pin should be wired to its corresponding pin on the device.
 
-The RS232 bus can also be used through the `MXP expansion port`_.
+The RS232 bus can also be used through the MXP Expansion Port.
 
-USB
-^^^
+USB Client
+^^^^^^^^^^
+
+.. image:: images/roborio-ports/roborio-usb-client.png
+
+One of the USB ports on the roboRIO is a USB-B, or USB client port.  This can be connected to devices, such as a driver station computer, with a standard USB cable.
```

Comment:
```suggestion
One of the USB ports on the roboRIO is a USB-B, or USB client port.  This can be connected to devices, such as a Driver Station computer, with a standard USB cable.
```